.circleci/config.yml

```
@@ -64,11 +58,25 @@ workflows:
   version: 2
   e2e:
     jobs:
-      - e2e-minikube
+      - e2e-k8s:
+          matrix:
+            parameters:
+              # https://kubernetes.io/releases
+              kubernetes-version:
+                - "v1.26.0"
```

Looks like it's [failing](https://app.circleci.com/pipelines/github/mamercad/stackstorm-k8s/9/workflows/9e65320a-bfcf-45f1-b6f4-730bf596e5c9/jobs/16) on K8s v1.26.0: 
```
! Specified Kubernetes version 1.26.0 is newer than the newest supported version: v1.25.3.
Use `minikube config defaults kubernetes-version` for details.
```
It was just released a month ago.

Let's perhaps swap the version?
```suggestion
                - "v1.23.15"
```

---

Sounds good, pushed!